src/page/Home.tsx

```diff
@@ -15,20 +17,132 @@ class Home extends React.Component<{}>
 
     public render() {
         return (
-            <div className="text-center">
-                <Row>
-                    <Col lg="8" className="m-auto">
-                        <SearchBox
-                            onChange={this.onSearch}
-                            placeholder="Search variants"
-                        />
-                    </Col>
-                </Row>
-                <Row className="py-4">
-                    <Col className="m-auto">
-                        {this.inputText}
-                    </Col>
-                </Row>
+            <div>
```

Comment:
This render method is a bit too long, can we have multiple methods/components here?

Reply:
do you mean make each row as a component?

Reply:
That would probably be too many components. Maybe for now just separate the Examples as another component.